barretenberg/cpp/src/barretenberg/ultra_honk/ultra_composer.hpp

```diff
@@ -29,8 +29,6 @@ template <UltraFlavor Flavor> class UltraComposer_ {
     static constexpr size_t num_zero_rows = Flavor::has_zero_row ? 1 : 0;
     static constexpr std::string_view NAME_STRING = "UltraHonk";
     static constexpr size_t NUM_WIRES = CircuitBuilder::NUM_WIRES;
-    std::shared_ptr<ProvingKey> proving_key;
```

Comment:
These were not used in any of our existing code paths (always null) and were causing confusion as I was debugging.